### PR TITLE
US-3382 Adjusted capabilities to reflect changes

### PR DESCRIPTION
**Pull request by pfoster-youitv**
Adjusted capabilities to reflect changes request from Appium
Removed client-side unsupported request handling
Added tests to prove the exceptions for unsupported By strategies come from the server
Re-enabled the tests that were failing on Android

### CODE REVIEW

File: src/test/java/io/appium/java_client/YouiEngine/SanityTest.java [hunk 1]
```diff
@@ -539,9 +544,127 @@ public void clickOnCyiTextAtlasTest() throws Exception {
         }
     }
 
+    // TODO update these to reflect the correct error
```
Comment (HeloiseRD) [inline src/test/java/io/appium/java_client/YouiEngine/SanityTest.java]: Maybe make notes here that these methods haven't been implemented yet and that is what you are testing?

**Comment by HeloiseRD** [comment]
Some comments describing why you are testing unsupported methods would be great.
Otherwise +1

**Comment by pfoster-youitv** [comment]
Oh - they're tested to make sure we handle attempts to use unsupported options. We don't want to crash the server or have execution go into limbo, we want a 501-type response.

File: src/test/java/io/appium/java_client/YouiEngine/SanityTest.java (hunk 1 above)
Comment (pfoster-youitv) [inline src/test/java/io/appium/java_client/YouiEngine/SanityTest.java]: Oh - they're tested to make sure we handle attempts to use unsupported options. We don't want to crash the server or have execution go into limbo, we want a 501-type response.

**Pull request merged**